src/patternfly/components/Pagination/pagination-menu-toggle.hbs

```diff
@@ -0,0 +1,12 @@
+{{#> menu-toggle
+  menu-toggle--IsPlain=true
+  menu-toggle--IsText=true
+  menu-toggle--modifier=pagination-menu-toggle--modifier menu-toggle--id=pagination-menu-toggle--id
+  menu-toggle--attribute='aria-label="Items per page"'}}
```

Comment:
@thatblindgeye nice, that `aria-label` was borrowed from the old options menu - took it out.

re: https://github.com/patternfly/patternfly/issues/6094, at least one difference I'd love to see for v6 (since it's breaking) is a `.pf-[v]-c-pagination__per-page-menu` (or something named better than that anyways!) to hold that menu/menu-toggle and we show/hide that element instead. wdyt? Otherwise it's probably the same update. WDYT?